### ISSUE TITLE
Want to connect to you

### ISSUE THREAD

**Issue by fitbyprabhat** [neonbjb/tortoise-tts#391]
Hey neon i want to be someday like you if there is any chance of being your assistant let me i really wanna work with you 🥺ᕙ⁠(⁠ ⁠~⁠ ⁠.⁠ ⁠~⁠ ⁠)⁠ᕗ

**Comment by neonbjb** [comment]
Hey Prabhat - thanks, it means a lot. The only way I would consider something like this is with someone local, though.. I just don't think it would be very productive for you otherwise.

- James

**Comment by fitbyprabhat** [comment]
Yes i do understand all the things I wish someone like you could be my mentor i am really messed up. I don't know how i can become like you from where to start.
 You are like God to me when it comes to development and research

**Comment by neonbjb** [comment]
It's not hard at all! 
1. Pick a problem you're interested in. Ideally something you really care about.
2. Carve out a few hours of time every day to work on that personal project.
3. Don't assume anything is over your head or outside of your abilities. You don't have to understand every aspect about a technique or technology to use it. That's just as true with ML as anything else.
4. ML / research is a bit different from a lot of work because you're going to be wrong a lot. Be ready for it to happen, and be ready to pivot or try something new.

Re-reading all this it sounds like chatgpt wrote it. I promise it didn't lol.
